### DISCUSSION TITLE
Generate dependencies diagram from adfInstance object type

### DISCUSSION THREAD

**Discussion by djdimmen** [Azure-Player/azure.datafactory.tools#410] [Q&A]
I was looking through the documentation and saw that the Get-AdfDocDiagram works when an object type of Adf is provided, but it will not work with the adfInstance object type. Is there a way to generate a diagram from an adf instance that is not connected directly to a git repository?

**Comment by NowinskiK** [comment]
No, that scenario is not supported.
